client/src/components/ChangePasswordModal.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <v-dialog width="500px">
     <v-card class="bg-backgroundGrey">
-      <v-img src="/images/Footer-min.jpeg" width="100%" max-height="16" cover />
+      <v-img src="/images/Footer-min.jpeg" width="100%" max-height="16" alt="" cover/>
```

Comment:
I would still give this a name, theres a few other places where the alt is empty

Reply:
![image](https://github.com/codersforcauses/csf/assets/91424496/1bab88e4-d444-4397-8a77-9bea17ffa2a8)

Reply:
I am trying to fix this error, do you know how to?

Reply:
![image](https://github.com/codersforcauses/csf/assets/91424496/c521804e-404d-4610-aa80-5ebe6576229e)
Package.json already seems to have it